pkg/dev/dev.go

```diff
@@ -181,22 +181,30 @@ outer:
 		}
 
 		opts.Build.Args = params
-		image, err := client.AcornImageBuild(ctx, file, &opts.Build)
-		if err != nil {
-			logrus.Errorf("Failed to build %s: %v", file, err)
-			logrus.Infof("Build failed, touch [%s] to rebuild", file)
-			go func() {
-				time.Sleep(120 * time.Second)
-				watcher.Trigger()
-			}()
-			continue
+		var image *v1.AppImage
+		if !opts.Build.Attach {
+			image, err = client.AcornImageBuild(ctx, file, &opts.Build)
+			opts.Build.ImageID = image.ID
+			if err != nil {
+				logrus.Errorf("Failed to build %s: %v", file, err)
+				logrus.Infof("Build failed, touch [%s] to rebuild", file)
+				go func() {
+					time.Sleep(120 * time.Second)
+					watcher.Trigger()
+				}()
+				continue
+			}
 		}
 
+		opts.Build.Attach = false
```

Comment:
Can we add a comment here explaining why this is getting set to false?

Reply:
added